Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------

fig = plt.figure(figsize=(12, 8), facecolor = "white")
ax = fig.add_axes([.1, 1, 1, 1])

omit = ['right', 'top']

sns.regplot(features[0], features[1], data = data01, ci = 90, color = "#353433", order = 2, label = "90%", scatter = False)
sns.regplot(features[0], features[1], data = data01, ci = 95, color = "#545454", order = 2, label = "95%", scatter = False)
sns.regplot(features[0], features[1], data = data01, ci = 99, color = "#9b9b9b", order = 2, label = "99%", scatter = False)
sns.regplot(features[0], features[1], data = data01, ci = 0, color = "black", order = 2, label = "", scatter = False)

ax.scatter(features[0], features[1], data = data01, s = 9, color = "#808080", label = "")

ax.legend(title = "Confidence intervals", loc = (.85, .1), prop={'size': 14})
ax.spines[omit].set_visible(False)
ax.set_ylabel(r'$\widehat{s}$', size = 19)
ax.set_xlabel(r'$\widehat{m}$', size = 19)
plt.show();
# plt.savefig("../figs/03scatter_01.png")
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[3], line 6[0m
[1;32m      2[0m ax [38;5;241m=[39m fig[38;5;241m.[39madd_axes([[38;5;241m.1[39m, [38;5;241m1[39m, [38;5;241m1[39m, [38;5;241m1[39m])
[1;32m      4[0m omit [38;5;241m=[39m [[38;5;124m'[39m[38;5;124mright[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mtop[39m[38;5;124m'[39m]
[0;32m----> 6[0m [43msns[49m[38;5;241;43m.[39;49m[43mregplot[49m[43m([49m[43mfeatures[49m[43m[[49m[38;5;241;43m0[39;49m[43m][49m[43m,[49m[43m [49m[43mfeatures[49m[43m[[49m[38;5;241;43m1[39;49m[43m][49m[43m,[49m[43m [49m[43mdata[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[43mdata01[49m[43m,[49m[43m [49m[43mci[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;241;43m90[39;49m[43m,[49m[43m [49m[43mcolor[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43m#353433[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43morder[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;241;43m2[39;49m[43m,[49m[43m [49m[43mlabel[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43m90[39;49m[38;5;124;43m%[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43mscatter[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;28;43;01mFalse[39;49;00m[43m)[49m
[1;32m      7[0m sns[38;5;241m.[39mregplot(features[[38;5;241m0[39m], features[[38;5;241m1[39m], data [38;5;241m=[39m data01, ci [38;5;241m=[39m [38;5;241m95[39m, color [38;5;241m=[39m [38;5;124m"[39m[38;5;124m#545454[39m[38;5;124m"[39m, order [38;5;241m=[39m [38;5;241m2[39m, label [38;5;241m=[39m [38;5;124m"[39m[38;5;124m95[39m[38;5;124m%[39m[38;5;124m"[39m, scatter [38;5;241m=[39m [38;5;28;01mFalse[39;00m)
[1;32m      8[0m sns[38;5;241m.[39mregplot(features[[38;5;241m0[39m], features[[38;5;241m1[39m], data [38;5;241m=[39m data01, ci [38;5;241m=[39m [38;5;241m99[39m, color [38;5;241m=[39m [38;5;124m"[39m[38;5;124m#9b9b9b[39m[38;5;124m"[39m, order [38;5;241m=[39m [38;5;241m2[39m, label [38;5;241m=[39m [38;5;124m"[39m[38;5;124m99[39m[38;5;124m%[39m[38;5;124m"[39m, scatter [38;5;241m=[39m [38;5;28;01mFalse[39;00m)

[0;31mTypeError[0m: regplot() got multiple values for argument 'data'

